### DISCUSSION TITLE
Question: SQL errors in production mode

### DISCUSSION THREAD

**Discussion by un-versed** [adonisjs/core#3788] [Help]
Hi!

How to prevent SQL errors in response to client?

![image](https://user-images.githubusercontent.com/12012284/177781052-2ff3efa1-4676-4084-9a85-ddf563065c95.png)

I'm getting SQL errors in the request response... There's no default handling for production mode? 

If I remember correctly, in adonis 4 there was some handling for production mode

For now I'm using this:

![image](https://user-images.githubusercontent.com/12012284/177784258-5a163515-f78c-43a7-bda0-2af33b88da21.png)

Every 500's errors sends the same message to client... I'm going to report other errors with a monitoring tool

But I'm still wondering if there is another solution

**Comment by thetutlage** [comment]
There is default handling and only the error message is reported to the client. If you do not want that, you can self handle exceptions as you are doing in the shared screenshot
